packages/react-core/src/demos/CustomMenus/CustomMenus.md

```diff
@@ -27,78 +27,56 @@ import styles from '@patternfly/react-styles/css/components/Menu/menu';
 
 ## Demos
 
-Custom menus can be constructed using a composable approach by combining the [Menu](/components/menus/menu) and [Menu toggle](/components/menus/menu-toggle) components in unique ways. Composable menus currently require consumer keyboard handling and use of our undocumented [popper.js](https://popper.js.org/) wrapper component called Popper. We understand this is inconvientent boilerplate and these examples will be updated to use [Dropdown](/components/dropdown) in a future release.
+Custom menus can be constructed using a composable approach by combining the [Menu](/components/menus/menu) and [Menu toggle](/components/menus/menu-toggle) components in unique ways. [Dropdown](/components/menus/dropdown), [Select](/components/menus/select), or [MenuContainer](/components/menus/menu#menucontainer) may be used in combination with menu components to handle basic keyboard inputs, or menu components may be connected manually through our undocumented internal [popper.js](https://popper.js.org/) wrapper component called Popper.
 
-### Composable simple dropdown
+### Actions menu
 
-```ts file="./examples/ComposableSimpleDropdown.tsx"
-```
-
-### Composable actions menu
-
-```ts file="./examples/ComposableActionsMenu.tsx"
-```
+```ts file="./examples/ActionsMenuDemo.tsx"
 
-### Composable simple select
-
-```ts file="./examples/ComposableSimpleSelect.tsx"
 ```
 
-### Composable simple checkbox select
-
-```ts isBeta file="./examples/ComposableSimpleCheckboxSelect.tsx"
-```
+### Favorites menu
 
-### Composable typeahead select
+```ts file="./examples/FavoritesDemo.tsx"
 
-```ts isBeta file="./examples/ComposableTypeaheadSelect.tsx"
 ```
 
-### Composable multiple typeahead select
-
-```ts isBeta file="./examples/ComposableMultipleTypeaheadSelect.tsx"
-```
+### Drilldown menu
 
-### Composable drilldown menu
+```ts isBeta file="./examples/DrilldownMenuDemo.tsx"
 
-```ts isBeta file="./examples/ComposableDrilldownMenu.tsx"
 ```
 
-### Composable tree view menu
+### Tree view menu
 
 When rendering a menu-like element that does not contain MenuItem components, [Panel](/components/panel) allows more flexible control and customization.
 
-```ts file="./examples/ComposableTreeViewMenu.tsx"
+```ts file="./examples/TreeViewMenuDemo.tsx"
+
 ```
 
-### Composable flyout
+### Flyout menu
 
 The flyout will automatically position to the left or top if it would otherwise go outside the window. The menu must be placed in a container outside the main content like Popper, [Popover](/components/popover) or [Tooltip](/components/tooltip) since it may go over the side nav.
```

Comment:
```suggestion
The flyout will automatically position to the left or top if it would otherwise go outside the window. The menu must be placed in a container outside the main content like Popper, [popover](/components/popover) or [tooltip](/components/tooltip), since it may go over the side nav.
```